matrix/tests/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    mat_ops: tests for matrix operations
```

Comment:
Please consider adding a carriage return at the end of this file.  It is a good idea to have a carriage return as the last line of your Python file but my real interest is beyond that.  This change will kick off a new Travis CI test and pytest is now a part of that testing and I am interested to see Travis finds, runs, and passes your tests.